fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -3,11 +3,10 @@
 
 import click
 from fideslog.sdk.python.utils import OPT_OUT_COPY
-import requests
 import toml
 
-from fidesctl.cli.utils import handle_cli_response, with_analytics
-from fidesctl.core import api as _api
+import fidesctl
+from fidesctl.cli.utils import with_analytics, check_server
```

Comment:
```suggestion
from fidesctl.cli.utils import check_server, with_analytics
```

It's a nit pick, but I know as soon as I Cmd+S this file locally, my VSCode settings are going to create this diff.